MoonDawgArduino/Libraries/Movement/Movement.h

```diff
@@ -1,25 +1,33 @@
-// Cameron Schwartzberg
-// 2/18/2024, 7:37 PM
-// 240216-212839-uno
-// Header file that contains all the classes and defines all functions for the "Movement.cpp"
-// Cameron Schwartzberg, Marco Caliendo, Andrew Barnes, Nadia Lafontant, Mitchell Wettig
+/*
+* This is the library the houses all of the functions and classes for moving and operating the bot
+* This file is the header of the library which has function declarations and classes
+* This file is connected to "Movement.cpp"
+* Owner: Cameron Schwartzberg
+* Contributors: Marco Caliendo, Andrew Barnes, and Mitchell Wettig
+* Date Created: 2/18/2024
+*/
 
 #ifndef Movement_h
 #define Movement_h
 
+// Calling all libraries 
 #include <stdint.h>
 #include <Servo.h>
 #include <Arduino.h>
 
 #include "PinDefinitions.h"
 
-
-// Class for the dig motor movement
-// Accesses the protected "Servo motor" and "(insert Actuator)" from the digSetup class
+/*
+* Class for the Dig movement
+* Private (things we don't want to be used by functions outside of this class):
+* This is the for using the Servo Library to create a controllable motor
+* Public:
+* This houses all of the functions that will be used by the main program

```

Comment:
a lot of this feels unnecessary, most people understand the purpose of public/private scope in classes

Reply:
Agree, esp since all of them are the same